www/js/app.js

```diff
@@ -153,11 +153,37 @@ function resizeIFrame () {
         }, 100);
     }
 }
+
+/**
+ * Resize the IFrame height, so that it fills the whole available height in the window
+ */
+function resizeLibrary () {
+    var headerStyles = getComputedStyle(document.getElementById('top'));
+    var iframe = document.getElementById('libraryIframe');
+
+    if (iframe.style.display === 'none') {
+        iframe.style.height = window.innerHeight + 'px';
+        var headerHeight = parseFloat(headerStyles.height) + parseFloat(headerStyles.marginBottom) + 10;
+        iframe.style.height = window.innerHeight - headerHeight + 'px';
+    } else {
+        // IE cannot retrieve computed headerStyles till the next paint, so we wait a few ticks
+        setTimeout(function () {
+            iframe.style.height = window.innerHeight + 'px';
+            var headerHeight = parseFloat(headerStyles.height) + parseFloat(headerStyles.marginBottom) + 10;
+            iframe.style.height = window.innerHeight - headerHeight + 'px';
+        }, 100);
+    }
+}
 document.addEventListener('DOMContentLoaded', function () {
     getDefaultLanguageAndTranslateApp();
     resizeIFrame();
+    resizeLibrary();
 });
 window.addEventListener('resize', resizeIFrame);
+window.addEventListener('resize', resizeLibrary);
```

Comment:
Never add two listeners for the same event. Instead, add a function to the event listener that branches the code, or runs `resizeIframe()` twice, once with the id of the ZIM iframe, and once with the id of the library iframe.